pytest/tests/sanity/validator_switch_key_quick.py

```diff
@@ -31,26 +31,51 @@ def test_validator_switch_key_quick(self):
             }
         }
 
+        default_stake = 50000000000000000000000000000000
+        stake_delta = 5000000000000000000000000000000
+
         # Key will be moved from old_validator to new_validator,
         # while the other_validator remains untouched.
         [
-            other_validator,
             old_validator,
+            other_validator,
             new_validator,
-        ] = start_cluster(2, 1, 3, None,
-                          [["epoch_length", EPOCH_LENGTH],
-                           ["block_producer_kickout_threshold", 10],
-                           ["chunk_producer_kickout_threshold", 10]],
-                          config_map)
-        wait_for_blocks(old_validator, count=5)
-
-        new_validator.reset_validator_key(other_validator.validator_key)
-        other_validator.kill()
+        ] = start_cluster(
+            2,
+            1,
+            3,
+            None,
+            [
+                ["epoch_length", EPOCH_LENGTH],
+                ["block_producer_kickout_threshold", 10],
+                ["chunk_producer_kickout_threshold", 10],
+                # Make `test0` always come earlier than `test1` in the validator
+                # set, so that they won't switch places because of different
+                # rewards.
+                # TODO(#12273): find better way to adjust validator stakes.
+                ["validators", 0, "amount",
+                 str(default_stake + stake_delta)],
+                ["validators", 1, "amount",
+                 str(default_stake - stake_delta)],
+                [
+                    "records", 0, "Account", "account", "locked",
+                    str(default_stake + stake_delta)
+                ],
+                [
+                    "records", 2, "Account", "account", "locked",
+                    str(default_stake - stake_delta)
+                ]
```

Comment:
nit: consider moving to a helper variable or method

Reply:
That's why I created #12273 :)